node/actors/internal_nats_server.go

```diff
@@ -62,9 +123,9 @@ func (ns *internalNatsServer) HandleMessage(from gen.PID, message any) error {
 			return err
 		}
 		ns.tokens[InternalNatsServerReadyName] = token
-		ns.Log().Info("registered publishable event %s, waiting for consumers...", InternalNatsServerReadyName)
+		ns.Log().Info("registered publishable event, waiting for consumers...", slog.Any("atom", InternalNatsServerReadyName))
```

Comment:
this is not an atom from a logging standpoint. This is an `event_name`

Reply:
This was accidental.